pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -148,6 +148,16 @@ type NetworkStats struct {
 	TxErrors *uint64 `json:"txErrors,omitempty"`
 }
 
+// NetworkStats contains data about network resources.
+type NetworkStats struct {
+	// The time at which these stats were updated.
+	Time metav1.Time `json:"time"`
+
+	InterfaceStats `json:",inline"`
```

Comment:
Please add a documentation line (`// Stats for the default interface, if found`)

Reply:
Will do.